backends/platform/sdl/sailfish/sdl-patches/0001-wayland-Bring-back-wl_shell-support.patch

```diff
@@ -0,0 +1,288 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
```

Comment:
Are there plans to upstream either of these patches?

Reply:
First patch is essentially rejected upstream but is maintained in sailfish upstream. I just wanted a bit fresher libSDL than sailfish provides in addition to my fix.
Second patch is already upstream but no release with it yet.

Reply:
So, we'll need a custom patched SDL for this. What are the effects of not having these patches included? (e.g. if a user decides to move to a newer vanilla SDL version)

Reply:
Without the first patch you'll get empty window (or rather no window at all). Without the second patch you'll lose mouse after first gesture within the window(minimizing window is fine) whether intended one or accidental

Reply:
And what do you mean by vanilla? Sailfish one already has first patch.

Reply:
> And what do you mean by vanilla? Sailfish one already has first patch.

I asked to clarify what would be the impact of installing a vanilla version of SDL, without either of these patches applied :)